lib/msal-common/src/app/module/AuthorizationCodeModule.ts

```diff
@@ -64,17 +72,73 @@ export class AuthorizationCodeModule extends AuthModule {
         // Populate query parameters (sid/login_hint/domain_hint) and any other extraQueryParameters set by the developer
         requestParameters.populateQueryParams();
 
-        return requestParameters.createNavigateUrl();
+        const urlNavigate = await requestParameters.createNavigateUrl();
+
+        // Cache token request
+        const tokenRequest: TokenExchangeParameters = {
+            scopes: requestParameters.scopes.getOriginalScopesAsArray(),
+            resource: request.resource,
+            codeVerifier: requestParameters.generatedPkce.verifier,
+            extraQueryParameters: request.extraQueryParameters,
+            authority: requestParameters.authorityInstance.canonicalAuthority,
+            correlationId: requestParameters.correlationId,            
+        };
+
+        this.cacheStorage.setItem(TemporaryCacheKeys.REQUEST_PARAMS, this.cryptoObj.base64Encode(JSON.stringify(tokenRequest)));
+
+        return urlNavigate;
     }
 
     async createAcquireTokenUrl(request: AuthenticationParameters): Promise<string> {
         throw new Error("Method not implemented.");
     }
 
+    async acquireTokenAuto(codeResponse: CodeResponse): Promise<TokenResponse> {
+        if (!codeResponse || !codeResponse.code) {
+            throw ClientAuthError.createAuthCodeNullOrEmptyError();
+        }
+
+        const encodedTokenRequest = this.cacheStorage.getItem(TemporaryCacheKeys.REQUEST_PARAMS);
+        const tokenRequest = JSON.parse(this.cryptoObj.base64Decode(encodedTokenRequest)) as TokenExchangeParameters;
+        tokenRequest.code = codeResponse.code;
+        tokenRequest.userRequestState = codeResponse.userRequestState;
+
+        this.cacheStorage.removeItem(TemporaryCacheKeys.REQUEST_PARAMS);
+        return this.acquireToken(tokenRequest);
+    }
+
     async acquireToken(request: TokenExchangeParameters): Promise<TokenResponse> {
-        throw new Error("Method not implemented.");
+        console.log(request);
+        return null;
```

Comment:
Mean to commit this?

Reply:
This function is implemented in the next PR, so I will leave it for now.